index.html

```diff
@@ -21,10 +54,12 @@ <h2>Add a New <span>Book</span></h2>
             <input class="input-title" id="title" type="text" placeholder="Title" required>
         </label>
         <label for="author">
-            <input class="input-author" id="author" type="text" maxlength="15" minlength="3" placeholder="Author" required>
+            <input class="input-author" id="author" type="text" maxlength="15" minlength="3" placeholder="Author"
+                required>
         </label>
         <button type="submit" class="add-btn">Add</button>
     </form>
     <script src="./js/classes.js"></script>
 </body>
+
```

Comment:
The **footer section is missing** from your pages.
- [ ] Kindly add a _footer section_ to your pages, which contains the text `Copyright`, and is aligned to the left of the _footer_.
- [ ] The _footer_ should **stick to the bottom** of the page.

Your sample Add New page:
![image](https://user-images.githubusercontent.com/12745474/138343594-6d120c16-845e-4daa-9234-62ffbe37ce0b.png)

**The wireframe pages:**
![image](https://user-images.githubusercontent.com/12745474/138343768-db7df029-2766-4f6c-b59a-67fd2ac3deb3.png)